### ISSUE TITLE
[Question] is there a way to change the look and feel of the workspaces?

### ISSUE THREAD

**Issue by simkimsia** [projectstorm/react-workspaces#59]
I have something prepared in figma and i have a team member who's more comfortable in NuxtJS/VueJS.

Is this something that my team member can change the look and feel while retaining react-workspaces as the main library?

**Comment by dylanvorster** [comment]
Yes, it would be best to look at the default package so you can see how the elements are pluggable. I don't have the capacity to share more info, but yes if you look through the code, everything is extensible by design. good luck!

**Comment by simkimsia** [comment]
i looked at the native JS demo and saw that emotion/styled is being used.

Essentially, I just need to use emotion/styled to restyle the components and that should work right?
